omod/src/main/java/org/openmrs/web/controller/PortletController.java

```diff
@@ -425,4 +465,30 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
 	protected void populateModel(HttpServletRequest request, Map<String, Object> model) {
 	}
 	
+	private Integer getStartIndexParameter(HttpServletRequest request) {
+		try {
+			return Integer.parseInt(request.getParameter("startIndex"));
```

Comment:
Can we also do the StringUtils.isNotBlank check for cases where startIndex may not be supplied?